x-pack/plugins/transform/public/app/hooks/use_delete_transform.tsx

```diff
@@ -188,7 +185,7 @@ export const useDeleteTransforms = () => {
           });
         }
         if (status.transformDeleted?.error) {
-          const error = extractErrorMessage(status.transformDeleted.error);
+          const error = status.transformDeleted.error.reason;
```

Comment:
Wouldn't it be safer to keep using `extractErrorMessage` since error formats have been changing?

Reply:
Actually, it seems like `extractErrorMessage` shouldn't be here in the first place, because it is not an HTTP error, but a simple object we set server-side with error details.

Reply:
I think `extractErrorMessage` was used because back then, error messages coming back from Kibana and Elasticsearch (and our own internal handling of the errors) were different in response shape. If we know for sure there will be only one type of error coming back for all 3 (transform, indexPattern, and index) then I think we don't have to use it.